ManatiCR/manatime
# 24: Tasks listing.

**Steps to test:**
- Create new project node.
- Create multiple tasks for testing and associates these to the new project.
- Check if the tasks are added in the project detaille.

**DoD**
- Go to a project.
- It should have a content pane view that lists all associated tasks (display should use panels).

[issue link](https://github.com/ManatiCR/manatime/issues/24)
